src/pages/missions.js

```diff
@@ -1,7 +1,34 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchMissions } from '../redux/missions/missions';
+import MissionRow from '../components/mission-row';
+import './missions.css';
 
-const Missions = () => (
-  <h1>Under Construction</h1>
-);
+const Missions = () => {
+  const missions = useSelector((state) => state.missions);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchMissions());
+  }, []);
+  return (
+    <section className="missions">
+      <table className="missions header">
+        <tr>
+          <th>Mission</th>
+          <th>Description</th>
+          <th>Status</th>
+          <th> </th>
+        </tr>
+        {missions.map((mission) => (
+          <MissionRow
+            name={mission.mission_name}
+            description={mission.description}
+            key={mission.mission_id}
+          />
+        ))}
+      </table>
```

Comment:
# Changes Required ♻️ 
- Good job so far. Everything is rendering just great. 
- There is only one problem, in the terminal there is an error as seen below. `<tr>` cannot appear as a child of `<table>` . Enclose it in a `<tbody>` and so on as mentioned in the terminal
![Screenshot (24)](https://user-images.githubusercontent.com/45460192/151057201-8cbcba20-85d9-47e8-9ce7-a7fa964f5cf9.png)